<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ChangesUi</class>
 <widget class="QDialog" name="ChangesUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="changeLA" >
     <property name="text" >
      <string>Change:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QTextBrowser" name="changeTB" />
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="previousPB" >
       <property name="toolTip" >
        <string>Go to previous change</string>
       </property>
       <property name="text" >
        <string>&amp;Previous change</string>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextPB" >
       <property name="toolTip" >
        <string>Go to next change</string>
       </property>
       <property name="text" >
        <string>&amp;Next change</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closePB" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="acceptPB" >
       <property name="toolTip" >
        <string>Accept this change</string>
       </property>
       <property name="text" >
        <string>&amp;Accept</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rejectPB" >
       <property name="toolTip" >
        <string>Reject this change</string>
       </property>
       <property name="text" >
        <string>&amp;Reject</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <includes>
  <include location="local" >qt_i18n.h</include>
 </includes>
 <tabstops>
  <tabstop>changeTB</tabstop>
  <tabstop>acceptPB</tabstop>
  <tabstop>rejectPB</tabstop>
  <tabstop>nextPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
